src/libraries/Common/tests/System/Net/Http/GenericLoopbackServer.cs

```diff
@@ -184,7 +184,8 @@ public class HttpRequestData
         public string Path;
         public Version Version;
         public List<HttpHeaderData> Headers { get; }
-        public int RequestId;       // Generic request ID. Currently only used for HTTP/2 to hold StreamId.
+        public int RequestId;       // HTTP/2 StreamId.
```

Comment:
Why don't we just make the `RequestId` `long` and reuse it?

Reply:
We could, but it would involve more significant changes. It seemed like more trouble than it was worth.